e2e/registerENSFlow.spec.js

```diff
@@ -0,0 +1,216 @@
+    }
+    await Helpers.checkIfVisible('wallet-screen', 80000);
+    await Helpers.enableSynchronization();
+  });
+
+  it('Should navigate to the Profile screen after swiping right', async () => {
+    await Helpers.swipe('wallet-screen', 'right', 'slow');
+    await Helpers.checkIfVisible('profile-screen');
+  });
+
+  it('Should navigate to Settings Modal after tapping Settings Button', async () => {
+    await Helpers.waitAndTap('settings-button');
+    await Helpers.checkIfVisible('settings-modal');
+  });
+
+  it('Should navigate to Developer Settings after tapping Developer Section', async () => {
+    await Helpers.waitAndTap('developer-section');
+    await Helpers.checkIfVisible('developer-settings-modal');
+  });
+
+  it('Should make ENS Profiles available', async () => {
+    await Helpers.tapByText('ENS Profiles');
+  });
+
+  it('Should show Hardhat Toast after pressing Connect To Hardhat', async () => {
+    await sendETHtoTestWallet();
+
+    await Helpers.waitAndTap('hardhat-section');
+    await Helpers.checkIfVisible('testnet-toast-Hardhat');
+  });
+
+  it('Should navigate to the Wallet screen after swiping left', async () => {
+    await Helpers.swipe('profile-screen', 'left', 'slow');
+    await Helpers.checkIfVisible('wallet-screen');
+  });
+
+  it('Should navigate to the Discover sheet screen after tapping Discover Button', async () => {
+    await Helpers.waitAndTap('discover-button');
+    await Helpers.checkIfVisible('discover-header');
+  });
+
+  it('Should go to ENS flow pressing the ENS banner', async () => {
+    await Helpers.waitAndTap('ens-register-name-banner');
+  });
+
+  it('Should be able to type a name that is not available', async () => {
+    await Helpers.checkIfVisible('ens-search-input');
+    await Helpers.typeText('ens-search-input', 'rainbowwallet', false);
+    await Helpers.delay(3000);
+    await Helpers.waitAndTap('ens-search-clear-button');
+  });
+
+  it('Should be able to type a name that is available and wait for fees', async () => {
+    await Helpers.checkIfVisible('ens-search-input');
+    await Helpers.typeText('ens-search-input', RANDOM_NAME, false);
+  });
+
+  it('Should be able to see network fees and name rent price', async () => {
+    await Helpers.checkIfVisible('ens-search-input');
+    await Helpers.checkIfVisible('ens-registration-fees');
+    await Helpers.checkIfVisible('ens-registration-price');
+  });
+
+  it('Should go to view to set records and skip it', async () => {
+    await Helpers.checkIfVisible('ens-search-continue-action-button');
+    await Helpers.waitAndTap('ens-search-continue-action-button');
+    await Helpers.checkIfVisible('ens-assign-records-skip');
+    await Helpers.waitAndTap('ens-assign-records-skip');
+  });
+
+  it('Should go to come back to records view and add some', async () => {
+    await Helpers.checkIfVisible('ens-confirm-register-sheet');
+    // await Helpers.swipe('ens-confirm-register-sheet', 'down', 'slow');
```

Comment:
are these commented lines going to be used later? or should we remove them

Reply:
yeah I want to add it when we have the entry points to continue registrations